pq-crypto/bike/decode.c

```diff
@@ -23,7 +23,7 @@
 #include "gf2x.h"
 
 // Decoding (bit-flipping) parameter
-#define MAX_IT 25
+#define MAX_IT 4

```

Comment:
The code in this file is the same as the already submitted pull request: https://github.com/awslabs/s2n/pull/1069